codespell_lib/data/dictionary.txt

```diff
@@ -55831,6 +55843,7 @@ thresolds->thresholds
 thresshold->threshold
 thressholds->thresholds
 thrid->third
+thridparty->thirdparty
```

Comment:
Not really a word, it's usually `third party` or `third-party`.

Reply:
I'll update that - I've seen that as a single word a lot of the time but that's probably 'european english'

Reply:
The [OED](https://www.oed.com/search/dictionary/?scope=Entries&q=third-party) has only `third party`, and [SCOWL (And Friends)](http://wordlist.aspell.net/) does not appear to be including `third-party` either. Other dictionaries such as [Cambridge](https://dictionary.cambridge.org/fr/dictionnaire/anglais/third-party) have both  `third party` and  `third-party`.

I really don't know. Would you like to suggest `thirdparty-> third party` only, or `thirdparty-> third party, third-party,`?

Reply:
I've added those suggestions.

Reply:
Ah, you have already chosen the latter. Great!